### ISSUE TITLE
Ongoing release - v1.1

### ISSUE THREAD

**Issue by kaolpr** [sinara-hw/SiLPA_HL#22]
@skuchars will be implementing fixes and improvements included in v1.1 milestone. If anyone has any suggestions or requests now is a final call.

**Comment by shanasz** [comment]
@skuchars when do You plan to push next commit? I need it for work on HE version.

**Comment by skuchars** [comment]
I've just made a commit.

**Comment by martafafara** [comment]
@skuchars when do you plan to close the current version?

**Comment by martafafara** [comment]
@gkasprow Do you know anything about progress?

**Comment by gkasprow** [comment]
I think the  design needs review

**Comment by martafafara** [comment]
Currenctly the design has unconnected nets, so layout still may change. I'm rather excepting some closed version

**Comment by gkasprow** [comment]
All is fine, those are copper balancing polygons and need to be added to the design rules as an Unrouted net

**Comment by martafafara** [comment]
I don't find traces in the pictures below  as the part of copper balancing.

![Image](https://github.com/user-attachments/assets/7952f9f1-0b69-4d67-ab88-010129833b13)

Additionally, the schematics are not consistent with PCB. Probably that is the reason why some issues implemented in schematic are not closed yet.

**Comment by gkasprow** [comment]
are you looking at devel 1.1branch?

**Comment by gkasprow** [comment]
but still schematic is not in sync @skuchars

**Comment by martafafara** [comment]
Yes, 1.1 branch with latest update 2 months ago. The main branch is closed
